components/LargeHeader.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+function LargeHeader() {
+  return (
+    <nav className="navbar navbar-default largeheader_background" style={{backgroundImage: 'url(' + 'assets/images/bg.png' +')'}}>
+        <div className="container-fluid">
+            <div className="navbar-header">
+                <button type="button" className="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-controls="navbar" aria-expanded="false">
+                    <span className="sr-only">Toggle navigation</span>
+                    <span className="icon-bar"></span>
+                    <span className="icon-bar"></span>
+                    <span className="icon-bar"></span>
+                </button>
+                <a className="navbar-brand" href="#">
+                    <img src={'assets/images/logo-small-white@3x.png'} />
+                </a>
+            </div>
+            <nav id="navbar" className="collapse navbar-collapse">
+            </nav>
+        </div>
+        <div id="largeheader_content" className="container">
+            <div className="row">
+                <div className="largeheader_centerText">
+                    Find the best community<br />
+                    resources for your needs
+                </div>
+            </div>
+            <div className="row">
+                <div className="col-lg-2 col-md-2 col-sm-1 col-xs-1">
```

Comment:
The bootstrap classes scale up. So if you had only used the class col-xs-12, it would use 12 columns for all screen views. As such, your classnames can be simplified here into "col-xs-1 col-md-2".